content-scripts/inject/module.js

```diff
@@ -91,6 +91,7 @@ const page = {
   },
   isFetching: false,
   async refetchSession() {
+    if (location.origin === "https://scratchfoundation.github.io") return;
```

Comment:
This function doesn't work when running scratch-www locally because the request is blocked by CORS. It should probably fetch `localhost:8333/session` instead of `scratch.mit.edu/session` when on `localhost:8333`.

Everything else works as expected.

Reply:
It could fetch `/session` instead of having a set domain.